site/app/controllers/admin/ReportController.php

```diff
@@ -589,6 +589,41 @@ private function getLateStatusMessage(LateDayInfo $ldi) {
         }
     }
 
+
+
+    #[Route("/courses/{_semester}/{_course}/reports/rainbow_grades_customization_save", methods: ["POST"])]
+    public function writetocustomization(): JsonResponse {
+        if (!empty($_SERVER['HTTP_X_REQUESTED_WITH']) && strtolower($_SERVER['HTTP_X_REQUESTED_WITH']) == 'xmlhttprequest') {
+            // Build a new model, pull in defaults for the course
+            $customization = new RainbowCustomization($this->core);
+            $customization->buildCustomization();
+
+            if (isset($_POST["json_string"])) {
+                try {
+                    $customization->processForm();
+                    return JsonResponse::getSuccessResponse(['status' => 'success']);
```

Comment:
The JSON Success response already defaults to having a success status, so you don't need to specify a 'status' here.